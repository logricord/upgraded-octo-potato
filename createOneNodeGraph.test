# Test creating a graph with one node

# Create a graph
CreateGraph graph1

# Add a node
AddNode graph1 node1

# Print the nodes in the graph
ListNodes graph1