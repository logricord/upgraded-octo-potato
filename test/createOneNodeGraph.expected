# Test creating a graph with one node

# Create a graph
created graph graph1

# Add a node
added node node1 to graph1

# Print the nodes in the graph
graph1 contains: node1